### PR TITLE
Added airsoft gear, updated paintball field itemgroups

### PR DESCRIPTION
**Pull request by bean-b**
#### Summary
Content "Added airsoft gear, updated paintball field itemgroups"

#### Purpose of change
The current paintball field only spawned paintballs, paintball guns and swim goggles. Swim goggles didint make much sense in terms of protective paint ball gear, and I figured the modular headgear system would be perfect, as plastic face shield, plastic mandible protector etc. would be perfect for this.

#### Describe the solution
A fancy miltiary headgear didint make much sense, so I added an airsoft headgear, which is a plastic headgear that has military rail mounting capabilities. I also added airsoft gloves. I also changed the spawn groups of the paintball field to spawn more protective gear, hopefully making it a bit more intersting to loot. 

Also made the balistic/plastic mandible protectors just a little less encumbering, because mouth encumbrance is really rough, and they didn't seem like they needed to be that high, especialy since they dont have great protection values

#### Describe alternatives you've considered

It's a little strange to see all the airsoft gear in the paintball arena, but it makes more sense then seeing swim goggles. 
There are more airsoft/paintball gear that could be added, such as steel mandible protector etc.
I think it would be cool to have a seperate airsoft arena, but that seemed out of the scope of this pr

#### Testing
Opened the items in game, teleported to a paintball arena, and checked the spawns

#### Additional context
gloves : https://www.etsy.com/listing/974154578/tactical-hard-knuckle-half-finger-gloves
helmet without attachments : https://www.evike.com/products/38914/%5D
helmet with attachemnts: https://toptacticalgear.com/products/anti-fog-airsoft-full-face-mask-helmet

### CODE REVIEW

File: data/json/itemgroups/Locations_MapExtras/locations_commercial.json [hunk 1]
```diff
@@ -894,7 +894,45 @@
...
+          { "item": "paintball", "prob": 125 },
+          { "item": "paintballgun", "prob": 100, "charges": [ 0, 50 ] },
+          { "group": "paintball_cloths", "prob": 30 }
+        ]
+      }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "paintball_cloths",
```
Comment (MNG-cataclysm) [inline data/json/itemgroups/Locations_MapExtras/locations_commercial.json:909]:
```suggestion
    "id": "paintball_clothes",
```
"Cloths" seems like an out-of-place name, at least to me.

File: data/json/itemgroups/Locations_MapExtras/locations_commercial.json [hunk 2]
```diff
@@ -894,7 +894,45 @@
     "type": "item_group",
     "id": "paintball_items",
     "//": "For paintball_field.json",
-    "items": [ [ "paintballgun", 55 ], [ "paintball", 45 ], [ "goggles_swim", 15 ] ]
+    "items": [
+      {
+        "distribution": [
+          { "item": "paintball", "prob": 125 },
+          { "item": "paintballgun", "prob": 100, "charges": [ 0, 50 ] },
+          { "group": "paintball_cloths", "prob": 30 }
```
Comment (MNG-cataclysm) [inline data/json/itemgroups/Locations_MapExtras/locations_commercial.json:902]:
```suggestion
          { "group": "paintball_clothes", "prob": 30 }
```

**Review by MNG-cataclysm — CHANGES_REQUESTED** [review]

**Review by MNG-cataclysm — APPROVED** [review]

**Comment by I-am-Erk** [comment]
because of the 'fluff' nature of this I think I'll put in freeze for now.

File: data/json/items/armor/head_attachments.json [hunk 3]
```diff
@@ -61,9 +61,9 @@
     "material": [ "thermo_resin" ],
     "symbol": "[",
     "color": "dark_gray",
-    "material_thickness": 1,
+    "material_thickness": 1.5,
     "flags": [ "HELMET_MANDIBLE_GUARD", "CANT_WEAR" ],
-    "armor": [ { "encumbrance": 12, "coverage": 100, "covers": [ "mouth" ] } ]
+    "armor": [ { "encumbrance": 10, "coverage": 100, "covers": [ "mouth" ] } ]
```
Comment (Fris0uman) [inline data/json/items/armor/head_attachments.json:66]: Not sure it needs to be less encumbering *and* more protective. This feels out of place for this PR, if you want to rebalance this gear you should probably do it in a different PR dedicated to this. Same for the two other similar buffs

**Pull request merged**